src/data/RaffleData.ts

```diff
@@ -1,6 +1,99 @@
 export const RAFFLE_DATA = [
   {
     id: 1,
+    title: 'OrangeHare presents KOREAN POP-REVOLUTION',
+    slug: 'orangehere-raffle',
+    body: 'Korean Pop-Revolution is an NFT collection of 2000 NFTs curated by a team of 8 Korean pop artists active in the Korean art scene across various fields such as design, fashion, illustration, oriental painting, fine art, and web3',
+    madeBy: 'OrangeHare',
+    imageUrl: '/images/orange-here.png',
+    details: [
+      {
+        address: '0x21d6bd949118bf92f87dcf0aa5911fa26f0a007f',
+        qty: 1,
+      },
+      {
+        address: '0x09e3d5d3bc9aa3b9ad4e8fd6568ba81b0cfd5ccc',
+        qty: 1,
+      },
+      {
+        address: '0xecac3e374e3dd1beef5734d8840673f7e7adab3a',
+        qty: 1,
+      },
+      {
+        address: '0x6c5fa9bf0e40731fb09def88ccf74f1f688690f4',
+        qty: 1,
+      },
+      {
+        address: '0xaca39b187352d9805deced6e73a3d72abf86e7a0',
+        qty: 1,
+      },
+      {
+        address: '0xe81eced4d1056a58d38a05a485bb43d50378e416',
+        qty: 1,
+      },
+      {
+        address: '0x09e3d5d3bc9aa3b9ad4e8fd6568ba81b0cfd5ccc',
+        qty: 1,
+      },
+      {
+        address: '0xfc762d68fd6c8674cda4532fedf48da7aeb2bd5c',
+        qty: 1,
+      },
+      {
+        address: '0xbc0e0e284610fe9c22ad177df68120b7323e3270',
+        qty: 1,
+      },
+      {
+        address: '0x6f0f535da243e441352a9040e556f976b935453b',
+        qty: 1,
+      },
+      {
+        address: '0x640a11d9a4b1db94b30a9cca25dc1e6870d7afc3',
+        qty: 1,
+      },
+      {
+        address: '0xd8ff659b400d546bb7c6f2dcb2193d36de91ce7b',
+        qty: 1,
+      },
+      {
+        address: '0x9996f0625a29ac20e06bac3e95e753b5bab2a6f6',
+        qty: 1,
+      },
+      {
+        address: '0x594b7efdda12093e98945c89243b71ff79409e4a',
+        qty: 1,
+      },
+
+      {
+        address: '0xdf23095637524f841a33372427c0262df3de6348',
+        qty: 1,
+      },
+      {
+        address: '0xd8ff659b400d546bb7c6f2dcb2193d36de91ce7b',
```

Comment:
group addresses. some addresses have more than 1 nft. make sure total is 20 and double check numbers to avoid having wrong data in the raffle resolution

Reply:
@kesar, the information I have doesn't contain the quantity each user has.

Reply:
cool sorry data wasnt very understandable 👍🏻